distribution/src/bin/elasticsearch.bat

```diff
@@ -68,6 +81,14 @@ IF "%checkpassword%"=="Y" (
   )
 )
 
+IF "!enrolltocluster!"=="Y" (
```

Comment:
I did some manual testing and the batch script seems to now be almost functional ( wrt to parsing parameters, picking up the enrollment token and passing a sanitized parameter list ( newparams ) to Elasticsearch. 

What currently doesn't work is 
- This CALL here triggers a "multiple garbage collectors defined" error message
- I know of no way to pass the keystore password as stdin to this execution. We can call the CLI batch script directly instead of this invocation if it helps ( I couldn't figure out how ) 

@rjernst , @williamrandolph I assume you are our resident windows batch file experts so calling upon you for any ideas

Reply:
I think I resolved this with the latest change. Still open to suggestions though